.github/workflows/ci.yml

```
@@ -14,9 +14,8 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-        - '2.7'
-        - '3.5'
-        - '3.7'
+        - '3.6'
+        - '3.8'
```

@jpivarski is there a need to drop Python 3.7 here as well? If not, might as well keep it.

```suggestion
        - '3.7'
        - '3.8'
```